backend/modules/parsers/multimodalparser.py

```diff
@@ -39,7 +43,22 @@ def arrayToBase64(image_arr: np.ndarray):
 
 class MultiModalParser(BaseParser):
     """
-    MultiModalParser is a multi-modal parser class for deep extraction of pdf documents
+    MultiModalParser is a multi-modal parser class for deep extraction of pdf documents.
+
+    Parser Configuration will look like the following while creating the collection:
+    {
+        "chunk_size": 1000,
+        "chunk_overlap": 20,
+        "parser_map": {
+            ".pdf": "MultiModalParser",
+        },
+        "additional_config": {
+            "model_configuration": {
+                # <provider>/<model_name> from model_config.yaml
+                "name": "truefoundry/openai-main/gpt-4-turbo"
+            }
+        }
+    }
```

Comment:
This can get messy fast

I would prefer to design it like so

```
"parser_map": {
  ".pdf":  {
    "name": "MultiModalParser",
    "config": {
      // any arguments to pass to MultiModalParser init as kwargs
     }
}
```

Reply:
Although I understand this would take more changes because of our existing structure

Reply:
Agree. We can add this as modification later. Where within config we have chunk size, chunk map and additional things. I'll add this as task